src/components/CompanionWindow.js

```diff
@@ -1,19 +1,91 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+import Paper from '@material-ui/core/Paper';
+import { withStyles } from '@material-ui/core/styles';
 import ns from '../config/css-ns';
+import WindowSideBarInfoPanel from '../containers/WindowSideBarInfoPanel';
+import WindowSideBarCanvasPanel from '../containers/WindowSideBarCanvasPanel';
 
 /**
  * CompanionWindow
  */
 class CompanionWindow extends Component {
+  /**
+   * activePanelComponent
+   * @return React Component
+   */
+  activePanelComponent() {
+    const { windowId, panelContent } = this.props;
+    switch (panelContent) {
+      case 'info':
+        return <WindowSideBarInfoPanel windowId={windowId} />;
+      case 'canvas_navigation':
+        return <WindowSideBarCanvasPanel windowId={windowId} />;
+      default:
+        return null;
+    }
+  }
+
   /**
    * render
    * @return
    */
   render() {
+    const {
+      classes, closeCompanionWindow, isDisplayed, position, t, windowId,
+    } = this.props;
     return (
-      <div className={ns('window-companion-side')} style={{ display: 'none' }} />
+      <Paper
+        className={[classes.root, ns(`companion-window-${position}`)].join(' ')}
+        style={{ display: isDisplayed ? null : 'none' }}
+        square={true} // eslint-disable-line react/jsx-boolean-value
```

Comment:
MUI has `false` by default so we need to set this to `true` (but eslint complains)

Reply:
Can you just set it like this?

```javascript
...
        style={{ display: isDisplayed ? null : 'none' }}
        square
...
```

Reply:
![🙈](https://thumbs.gfycat.com/NearThinGreatargus-size_restricted.gif)